src/CSSBattle/12_WigglyMoustache/Yongbin.html

```diff
@@ -0,0 +1,72 @@
+<div class="left"></div>
+<div class="mid"></div>
+<div class="right"></div>
+<div class="small1"></div>
+<div class="small2"></div>
+<style>
+  .left {
+    width: 60px;
+    height: 30px;
+    background: #f5d6b4;
+    border: solid #d86f45;
+    border-bottom-left-radius: 60px;
+    border-bottom-right-radius: 60px;
+    border-width: 20px;
+    border-top: 0;
+    position: fixed;
+    top: 150;
+    left: 70;
+  }
+  .mid {
+    width: 60px;
+    height: 30px;
+    background: #f5d6b4;
+    border: solid #d86f45;
+    border-bottom-left-radius: 60px;
+    border-bottom-right-radius: 60px;
+    border-width: 20px;
+    border-top: 0;
+    position: fixed;
+    top: 101;
+    left: 150;
+    transform: rotate(180deg);
+  }
+  .right {
+    width: 60px;
+    height: 30px;
+    background: #f5d6b4;
+    border: solid #d86f45;
+    border-bottom-left-radius: 60px;
+    border-bottom-right-radius: 60px;
+    border-width: 20px;
+    border-top: 0;
+    position: fixed;
+    top: 150;
+    right: 70;
+  }
+  .small1 {
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    background: #d86f45;
+    z-index: 2;
+    position: fixed;
+    left: 70px;
+  }
+  .small2 {
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    background: #d86f45;
+    z-index: 2;
+    position: fixed;
+    right: 70px;
+  }
+  body {
+    background: #f5d6b4;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
Not sure if it's convention, but consider moving `body` div to the top since it affects everything in the page

Reply:
Resolved in commit 5ec733b.